tutorials/test-prop-500/content/index.mdx

```diff
@@ -0,0 +1,103 @@
+---
+title: '{{title}}'
+keywords: ['kafe']
+date: '2020-04-24'
```

Comment:
Can we please update the date? xoxo

Reply:
Absolutely! I'm not one to live in the past.